agent/engine/docker_container_engine.go

```diff
@@ -848,3 +849,11 @@ func (dg *dockerGoClient) removeImage(imageName string) error {
 	}
 	return client.RemoveImage(imageName)
 }
+
+func (dg *dockerGoClient) LoadImage(opts docker.LoadImageOptions) error {
+	client, err := dg.dockerClient()
```

Comment:
Hmm, alright but we have a wide range of timeouts (30s - 2hrs). 
Do you have any estimates for a good timeout here ?